eng/jobs/linux-build.yml

```diff
@@ -22,9 +22,11 @@ jobs:
   - job: ${{ parameters.displayName }}
     displayName: ${{ parameters.displayName }}
     pool:
-      ${{ if eq(variables['System.TeamProject'], 'public') }}:
+      ${{ if eq(parameters.displayName,'Build_Linux_x64_FreeBSD') }}:
+        name: dnceng-freebsd-internal
```

Comment:
It seems bad that this is available in the public project for PR CI.

Reply:
yeah, looks like core-sdk too uses the same way. https://github.com/dotnet/core-sdk/commit/7c7f7bae6aa113c2b0bcf3b699dda6ecbf561e77 .

Reply:
I'll start a mail thread.